lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -546,9 +546,9 @@ class ReportUIFeatures {
    * @param {LH.Result} json
    */
   static openTreemap(json) {
-    const treemapDebugData = /** @type {LH.Audit.Details.DebugData} */ (
+    const treemapData = /** @type {LH.Audit.Details.TreemapData} */ (
       json.audits['script-treemap-data'].details);
-    if (!treemapDebugData) {
+    if (!treemapData) {
```

Comment:
ok, it won't let me do a suggestion across these lines, but don't need the cast here:
```js
    const treemapData = json.audits['script-treemap-data'].details;
    if (!treemapData || treemapData.type !== 'treemap-data') {
      throw new Error('no script treemap data found');
    }
```